dbt_copilot_helper/templates/addons/env/aurora-postgres.yml

```diff
@@ -151,3 +152,309 @@ Resources:
       FromPort: 5432
       ToPort: 5432
       SourceSecurityGroupId: !Ref EnvironmentSecurityGroup
+
+  # Enable ingress from Lambda Functions.
```

Comment:
I don't think we need these comments, the descriptions cover it already.

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/185/commits/a385eceb3ab4c605beb82f257a738a8b87386a51.